tutorials/plot_spatial.py

```diff
@@ -77,9 +80,10 @@
 ###################################################################################################
 
 # Plot positions (coordinates marked x are the actual points)
-plot_positions(position, alpha=1, ls='-', marker='x', c='tab:gray', markersize=10,
+plot_positions(position, alpha=1, ls='-', marker='x', color='tab:gray', markersize=10,
                title='Tracking', xlabel='x-position', ylabel='y-position')
 plt.legend(['coordinates'])
+plt.show()
```

Comment:
I don't think this is needed? The plots should show up in the rendered version without needing to call this.

Reply:
Because of the legend, without the `plt.show()` the plot outputs some image information. Is there a better way of suppressing this output?

Reply:
Ohhh. You should be able to squash that output by adding a semi-colon to the last plot call, so for example:
`plt.legend(['coordinates']);`

If that words, you won't need the `plt.show` call